js/ccf-app/src/crypto.ts

```diff
@@ -108,7 +108,7 @@ export const eddsaPemToJwk = ccf.crypto.eddsaPemToJwk;
 export const pubJwkToPem = ccf.crypto.pubJwkToPem;
 
 /**
- * @inheritDoc global!CCFCrypto.JwkToPem
+ * @inheritDoc global!CCFCrypto.jwkToPem
```

Comment:
These docs fixes are good, but can we put them in a separate PR?

Reply:
Sure, they were just drive by fixing of Sphinx warnings.

Reply:
#6120